docs/configure/settings.md

```diff
@@ -0,0 +1,47 @@
+# Settings
+
+Configuration settings are read from the `[tool.autopub]` section of the project’s `pyproject.toml` file, which you should add before customizing your configuration via the settings below.
```

Comment:
shall we add a link to the specs? https://www.python.org/dev/peps/pep-0518/#specification

Reply:
Good suggestion. Done!